logistics_planning_base/views/logistics_schedule_views.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="logistics_schedule_view_tree" model="ir.ui.view">
+        <field name="name">logistics.schedule.tree</field>
+        <field name="model">logistics.schedule</field>
+        <field name="arch" type="xml">
+            <tree editable="top">
+                <button
+                    name="action_logistics_schedule_form_view"
+                    icon="fa-gear"
+                    class="p-0"
+                    type="object"
+                    title="Edit"
+                />
+                <field name="partner_id" />
+                <field name="stock_move_id" domain="[('partner_id', '=', partner_id), ('product_id', '=', product_id), ('state', '=', 'done'), ('logistics_schedule_id', '=', False)]"/>
+                <field name="transport_type" />
+                <field name="product_id" />
+                <field name="product_uom_qty" />
+                <field name="scheduled_load_date" />
+                <field name="commitment_date" />
+                <field name="currency_id" invisible="1"/>
+                <field name="logistic_price_unit" widget="monetary" />
+                <field name="logistic_price_unit_done" widget="monetary" />
+                <field name="carrier_id" />
+                <field name="license_plate_1" />
+                <field name="license_plate_2" />
+                <field name="state" readonly="1"/>
+                <field name="note" />
+                <button
+                    name="%(action_logistics_schedule_cancel)d"
+                    icon="fa-ban"
+                    class="p-0"
+                    type="action"
+                    title="Cancel"
+                />
+            </tree>
+        </field>
+    </record>
+
+    <record id="logistics_schedule_view_search" model="ir.ui.view">
+        <field name="name">logistics.schedule.search</field>
+        <field name="model">logistics.schedule</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="picking_id"/>
+                <field name="partner_id"/>
+                <field name="product_id"/>
+                <separator/>
+                <filter string="Ground" name="transport_type_ground" domain="[('transport_type', '=', 'ground')]"/>
```

Comment:
Please add a filter for "Pending" schedules (those that are in `draft` or `ready` states). This filter should be enabled by default, so we can exclude those done or cancelled schedules.